src/main/controllers/JobTitleController.ts

```diff
@@ -25,7 +25,7 @@ export default class JobTitleController {
         label: (l: AnyRecord): string => l.jobTitle,
         hint: (l: AnyRecord): string => l.hint,
         attributes: {
-          autocomplete: 'organization-title',
+          autocomplete: 'organization',
```

Comment:
this looks like a typescript change - not quite sure why it was necessary to change this field, but who knows